charts/kyverno/values.yaml

```diff
@@ -231,23 +232,6 @@ config:
   - '[ReplicaSet,*,*]'
   - '[ReportChangeRequest,*,*]'
   - '[ClusterReportChangeRequest,*,*]'
-  # exclude resources from the chart
```

Comment:
Those were added originally by contribution from community to only exclude things added by Helm chart, rather than excluding entire namespace.  These are just redundant to the namespace exclusion so figured cleanup the more verbose exclusion for one targeted at just the namespace exclusion